core/src/main/java/cucumber/runner/Scenario.java

```diff
@@ -93,4 +96,10 @@ public Throwable getError() {
 
         return max(stepResults, Result.SEVERITY).getError();
     }
+
+    private void busSend(EmbedEvent embedEvent) {
+        if (bus != null) {
```

Comment:
I couldn't find any calls into the `Scenario` constructor where `bus` would be null.

Reply:
I removed the check for null, also did it in `write` method for consistency.